molgenis-data-i18n/src/main/java/org/molgenis/data/i18n/LocalizationService.java

```diff
@@ -163,15 +162,10 @@ public void addMissingMessageIDs(String namespace, Set<String> messageIDs)
 											   .map(key -> l10nStringFactory.create()
 																			.setMessageID(key)
 																			.setNamespace(namespace));
-			try
-			{
-				dataService.add(L10N_STRING, entities);
-				LOG.debug("Added message IDs to namespace '{}' : {}.", namespace, messageIDs);
-			}
-			catch (MolgenisPermissionException ex)
-			{
-				LOG.info("No permission to add message IDs to namespace '{}' : {}.", namespace, messageIDs, ex);
-			}
+
+			dataService.add(L10N_STRING, entities);
+			LOG.debug("Added message IDs to namespace '{}' : {}.", namespace, messageIDs);
+
```

Comment:
If we remove the catch, the method will suddenly begin to throw runtime exceptions it previously nommed up. Is this intentional?

Reply:
Ahum.... cannot remember removing this... will revert